atuin/src/command/client/stats.rs

```diff
@@ -16,7 +16,7 @@ use time::{Duration, OffsetDateTime, Time};
 #[derive(Parser, Debug)]
 #[command(infer_subcommands = true)]
 pub struct Cmd {
-    /// compute statistics for the specified period, leave blank for statistics since the beginning
+    /// Compute statistics for the specified period, leave blank for statistics since the beginning. Period can be either one of the following keywords: all, today, month, week, year. For a more fine-grained time interval, the period can be specified using any date format supported by the interim crate, see also https://docs.rs/interim/0.1.1/interim/#supported-formats.
```

Comment:
I'd rather make this a little shorter, and link to our own docs: https://docs.atuin.sh/reference/stats/

Reply:
Ohh, I wasn't aware of that doc! I'll update the PR, thanks for the review!